spec/android_update_release_notes_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::AndroidUpdateReleaseNotesAction do
+  describe '#android_update_release_notes' do
+    it 'adds a new section on RELEASE-NOTES.txt' do
+      in_tmp_dir do |tmp_dir|
+        # Arrange
+        ENV['PROJECT_ROOT_FOLDER'] = tmp_dir
+        release_notes_txt = File.join(tmp_dir, 'RELEASE-NOTES.txt')
+        File.write(release_notes_txt, ANDROID_FAKE_CONTENT)
+
+        # Act
+        run_described_fastlane_action(
+          new_version: '1.0'
+        )
+
+        # Assert
+        expect(File.read(release_notes_txt)).to eq(ANDROID_NEW_SECTION + ANDROID_FAKE_CONTENT)
+      end
+    end
+
+    it 'adds a new section on the given file' do
+      in_tmp_dir do |tmp_dir|
+        # Arrange
+        ENV['PROJECT_ROOT_FOLDER'] = tmp_dir
+        changelog_md = File.join(tmp_dir, 'CHANGELOG.md')
+        File.write(changelog_md, ANDROID_FAKE_CONTENT)
+
+        # Act
+        run_described_fastlane_action(
+          new_version: '1.0',
+          release_notes_file_path: changelog_md
+        )
+
+        # Assert
+        expect(File.read(changelog_md)).to eq(ANDROID_NEW_SECTION + ANDROID_FAKE_CONTENT)
+      end
+    end
+  end
+end
+
+ANDROID_FAKE_CONTENT = <<~CONTENT.freeze
```

Comment:
Prefer using a `let(:fake_content) { … }` to provide constant-like values in specs, rather than top-level constants (which will end up being publicly exposed/visible to the all rest of the codebase… as you realized in your last commit)

Reply:
PS: I know, `release_notes_helper_spec` does this as well… but that was written before we knew better and were better at Ruby and rspec 😛  (ideally we should change the ones in `release_notes_helper_spec.rb` as well)

Reply:
I said nothing 😆

Done in f0ced0647ed21ea0ca3d33440a8d8a7fda27bb41